### PR TITLE
Implement query() in FindMeetingQuery.java

### PR DESCRIPTION
**Pull request by ALai2**
Started week 5 walkthrough where query() in FindMeetingQuery.java needs to be implemented. The function passes all the given test cases when running the command "mvn test".

### CODE REVIEW

File: walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java [hunk 1]
```diff
@@ -15,9 +15,57 @@
...
+        if (eventAttendees.contains(attendee)) {
+          addEvent = true;
+          break;
+        }
+      }
+      
+      // add event if at least one of the attendees attend this event
+      if (addEvent) {
+        outOfRange.add(event.getWhen());
+      }
```
Comment (linxiz) [inline walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java]: is the boolean addEvent necessary? could you directly do this when you find eventAttendees.contains(attendee) and break the for loop?

**Review by linxiz — APPROVED** [review]
Do you want to add tests for the FindMeetingQuery?

**Comment by ALai2** [comment]
> Do you want to add tests for the FindMeetingQuery?

The tests that they gave us to check the functionality for mandatory attendees were really comprehensive so I don't think I would want to add tests to check mandatory attendees events. However, I do add tests for the functionality for optional attendees in a later commit.

File: walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java (hunk 1 above)
Comment (ALai2) [reply walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java]: Looking at my code again I can understand that the boolean addEvent isn't necessary. I added a commit which moves addEvent and instead adds the outOfRange.add(event.getWhen()) inside the if statement which checks eventAttendees.contains(attendee).

**Pull request merged**